File: test7.cpp
#include<iostream>
#include<cmath>

bool prime(int x)
{
  int y;
  for(y = 2; y <= sqrt(x); y++)
    if (x % y == 0)
      return false;
  return true;
}

int main()
{
  int N, tmp = 0, j = 0;

  std::cin >> N;

  for(int i = 0; i <= N; i++)
  {
    if(prime(i))
    {
      if(i - tmp == 2)
        j++;
      tmp = i;
    }
  }

  std::cout << j << std::endl;
  system("pause");
}

File: test7.py
import math

def prime(x):
  i = (int)(x ** 0.5)
  for y in range(2, i + 1, 1):
    if(x % y == 0):
      return False
  return True

N = eval(input())
tmp = 2
j = 0

for i in range(0, N + 1, 1):
  if(prime(i)):
    if(i - tmp == 2):
      j += 1
    tmp = i

print(j)